src/app/core/services/stream.service.ts

```diff
@@ -42,15 +42,15 @@ export function buildProgressStream(source: BinaryStream, onRead: (readBytes: nu
     },
     cancel() {
       return reader.cancel();
-    }
+    },
   });
 }
 
 export function joinReadableBinaryStreams(streams: BinaryStream[]): ReadableStream {
-  const streamsCopy = streams.map(s => s);
+  const streamsCopy = streams.map((s) => s);
```

Comment:
Would the spread synthax
```suggestion
  const streamsCopy = [...streams];
```